---
geometry_msgs/Point estimated_position
int32 estimated_class